Firmware/pins_Rambo_1_3.h

```diff
@@ -18,85 +30,107 @@
 
-#ifdef SNMM 
-  #define E_MUX0_PIN           17
-  #define E_MUX1_PIN           16
-  #define E_MUX2_PIN           84
-#endif
+#define HEATER_BED_PIN          4 //PG5
+#define TEMP_BED_PIN            2 //A2
 
-#ifdef DIS
-  #define D_REQUIRE            30
-  #define D_DATA               20
-  #define D_DATACLOCK          21
-#endif
+#define HEATER_0_PIN            3 //PE5
+#define TEMP_0_PIN              0 //A0
 
-// The SDSS pin uses a different pin mapping from file Sd2PinMap.h
-#define SDSS                   53
+#define HEATER_1_PIN           -1
+#define TEMP_1_PIN              1 //A1
 
-#ifndef SDSUPPORT
-// these pins are defined in the SD library if building with SD support
-  #define SCK_PIN              52
-  #define MISO_PIN             50
-  #define MOSI_PIN             51
-#endif
+#define HEATER_2_PIN           -1
+#define TEMP_2_PIN             -1
 
-#define BEEPER                 84
+#define TEMP_AMBIENT_PIN        6 //A6
 
-#define BTN_EN1                72
-#define BTN_EN2                14
-#define BTN_ENC                 9
+#define TEMP_PINDA_PIN          3 //A3
 
-#define SDCARDDETECT           15
+#define VOLT_PWR_PIN            4 //A4
+#define VOLT_BED_PIN            9 //A9
 
-#define LCD_PINS_RS            82
-#define LCD_PINS_ENABLE        18
-#define LCD_PINS_D4            19
-#define LCD_PINS_D5            70
-#define LCD_PINS_D6            85
-#define LCD_PINS_D7            71
 
 #define E0_STEP_PIN            34
 #define E0_DIR_PIN             43
 #define E0_ENABLE_PIN          26
 #define E0_MS1_PIN             65
 #define E0_MS2_PIN             66
-#define LED_PIN                13
 
-#ifdef THREEMM_PRINTER
-  #define FAN_PIN               8
-#else
-  #define FAN_PIN               6
-#endif
 
-#define KILL_PIN               -1 //80 with Smart Controller LCD
-#define SUICIDE_PIN            -1  //PIN that has to be turned on right after start, to keep power flowing.
-#define SDPOWER                -1
-#define HEATER_2_PIN           -1
+#define MOTOR_CURRENT_PWM_XY_PIN 46
+#define MOTOR_CURRENT_PWM_Z_PIN  45
+#define MOTOR_CURRENT_PWM_E_PIN  44
+#define SDPOWER             -1
+#define SDSS                53
+#define LED_PIN             13
+#define FAN_PIN              6
+#define FAN_1_PIN           -1
+#define PS_ON_PIN           -1
+#define KILL_PIN            -1  // 80 with Smart Controller LCD
+#define SUICIDE_PIN         -1  // PIN that has to be turned on right after start, to keep power flowing.
+
+#ifdef ULTRA_LCD
+
+//#define KILL_PIN            32
+
+#ifdef NEWPANEL
+
+#define BEEPER              84  // Beeper on AUX-4
+#define LCD_PINS_RS         82
+#define LCD_PINS_ENABLE     18
+#define LCD_PINS_D4	        19
+#define LCD_PINS_D5         70
+#define LCD_PINS_D6         85
+#define LCD_PINS_D7         71
+
+//buttons are directly attached using AUX-2
+#define BTN_EN1                72
+#define BTN_EN2                14
+#define BTN_ENC                 9  // the click
+
+#define SDCARDDETECT           15
+
```

Comment:
Missing TACH_0_PIN for the Noctua rpm sensor. As far i can see x-max, y-max and z-max are the last free pins can be used for that.
P1+P2 are used for LCD+Encoder+Beeper+SDcard
and P3 pins 3+5 are reserved for MMU
as well the P3 pins 6+8 for I2C (Filament Sensor)